sdk/core/core-util/review/core-util.api.md

```diff
@@ -10,6 +10,17 @@ export function delay(timeInMs: number): Promise<void>;
 // @public
 export const isNode: boolean;
 
+// @public
+export type ParsedWWWAuthenticate = {
+    [Key in ValidParsedWWWAuthenticateProperties]?: string;
+};
+
+// @public
+export function parseWWWAuthenticate(wwwAuthenticate: string): ParsedWWWAuthenticate;
```

Comment:
do we need these changes in this PR? I don't see `core-util` used (or even depended on) by core-rest-pipeline.

I'm fine stashing them here in general, but can they be their own PR if not needed for the feature support in this one?

Reply:
Not needed here. I will include them in a PR that uses them.

Reply:
Removed